javav2/example_code/dynamodb/src/main/java/com/example/dynamodb/DynamoDBPartiQ.java

```diff
@@ -0,0 +1,107 @@
+//snippet-sourcedescription:[DynamoDBPartiQ.java demonstrates how to work with PartiQL for Amazon DynamoDB.]
+//snippet-keyword:[AWS SDK for Java V2]
+//snippet-keyword:[Code Sample]
+//snippet-service:[Amazon DynamoDB]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[02/02/2022]
+//snippet-sourceauthor:[scmacdon-aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.dynamodb;
+
+// snippet-start:[dynamodb.java2.partiql.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.ExecuteStatementRequest;
+import software.amazon.awssdk.services.dynamodb.model.ExecuteStatementResponse;
+import java.util.ArrayList;
+import java.util.List;
+// snippet-end:[dynamodb.java2.partiql.import]
+
+/*
+ * Prior to running this code example, create an Amazon DynamoDB table named Music as discussed in this topic:
+ *
+ * https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ql-gettingstarted.html
+ *
+ * Also, ensure that you have set up your development environment, including your credentials.
+ *
+ * For information, see this documentation topic:

```

Comment:
For more information, see the following topic:

Reply:
fixed